openmc/material.py

```diff
@@ -641,37 +641,45 @@ def get_nuclide_atom_densities(self):
         nucs = []
         nuc_densities = []
         nuc_density_types = []
+
         for nuclide in nuclides.items():
             nuc, nuc_density, nuc_density_type = nuclide[1]
             nucs.append(nuc)
             nuc_densities.append(nuc_density)
             nuc_density_types.append(nuc_density_type)
 
+        nucs = np.array(nucs)
+        nuc_densities = np.array(nuc_densities)
+        nuc_density_types = np.array(nuc_density_types)
+
         if sum_density:
             density = np.sum(nuc_densities)
+
         percent_in_atom = np.all(nuc_density_types == 'ao')
         density_in_atom = density > 0.
         sum_percent = 0.
 
         awrs = []
-        for n, nuclide in enumerate(nuclides.items()):
+        for nuclide in nuclides.items():
             awr = openmc.data.atomic_mass(nuclide[0])
             if awr is not None:
-                awrs.append(awr / openmc.data.NEUTRON_MASS)
+                awrs.append(awr)
```

Comment:
Why did you remove the division by the neutron mass?   `openmc.data.atomic_mass` returns the mass in units of amu, but the awr is the atomic weight ratio, not amu.